questionnaires/vaping_weekly_survey/vaping_weekly_survey_armt.json

```diff
@@ -0,0 +1,856 @@
+[
+    {
+        "field_name": "cough",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item, please mark the response that best describes you currently.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "0 I never cough "
+            },
+            {
+                "code": "2",
+                "label": "2 "
+            },
+            {
+                "code": "3",
+                "label": "3 "
+            },
+            {
+                "code": "4",
+                "label": "4 "
+            },
+            {
+                "code": "5",
+                "label": "5 I cough all the time"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "cough_matrix",
+        "matrix_ranking": "",
+        "field_annotation": "range-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "phlegm",
+        "form_name": "vaping_weekly_survey",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "For each item, please mark the response that best describes you currently.",
```

Comment:
can we remove `For each item`? Similar for all the Questionnaires. but can confirm with the clinical team before making the change.

Reply:
requested Leonie take a look at this. I believe these are bespoke questionnaires rather than previously validated ones so changing the wording where sensible should be ok.

Reply:
Hi, yes, 'for each item' can be removed.